### PR TITLE
IBA-87 Add a custom property to yaml to mark internal endpoints

### PR DESCRIPTION
**Pull request by BullDog57Rus**

### CODE REVIEW

File: hub/hub-api.yaml [hunk 1]
```diff
@@ -472,15 +472,17 @@ paths:
                 $ref: "#/components/schemas/ResponseError"
 
   /internal/limitUsages:
+    x-access: private
```
Comment (aehrenthal) [inline hub/hub-api.yaml:475]: Do you think a boolean value might be better in terms of knowing what values are allowed/disallowed?

Comment (olyavertwist) [reply hub/hub-api.yaml:475]: @aehrenthal on the one hand, you are right but this approach will require us to apply the boolean property to ALL endpoints that would be a huge waste of time, IMO. It turns out currently just one endpoint should be private.

Comment (aehrenthal) [reply hub/hub-api.yaml:475]: Ok! But we can just assume it's false, unless its x-private: true. But either way is fine with me.

I think we also have all lead router endpoints @olyavertwist, no?

Comment (BullDog57Rus) [reply hub/hub-api.yaml:475]: for me both works. I just tried to use the same notation as java does: access levels of methods :D
@Zemidar maybe you have a better suggestion?

Comment (BullDog57Rus) [reply hub/hub-api.yaml:475]: oh, I had no idea about lead router endpoints. I would appreciate it if you @aehrenthal help me with that one :) (I mean if I have to mark them or not)

Comment (aehrenthal) [reply hub/hub-api.yaml:475]: Sure! The entire lead routing microservice is an internal service :)

Comment (BullDog57Rus) [reply hub/hub-api.yaml:475]: ah ok!

Comment (olyavertwist) [reply hub/hub-api.yaml:475]: Actually, the whole Lead Router yaml is skipped in our API docs entirely. But maybe for the sake of consistency, we can apply x-access property on the top level.

**Review by olyavertwist — APPROVED** [review]

**Pull request merged**